copter/source/docs/traditional-helicopter-tuning.rst

```diff
@@ -7,78 +7,13 @@ Traditional Helicopter – Tuning
-swashplate command. These are summed and sent to the mixing unit where the servo
-positions are determined.
-
-So this tuning method uses the FF gain initially to ensure the requested rates
-match the actual rates.  However the rates can vary from the requested due to
-disturbances. The P and D gains are then used to guard against disturbances
-that cause the actual rates to deviate from the requested rates. So the P and D
-gain may not be able to keep the actual rates exactly matching the requested
-rates.  Since the software is tracking where the orientation of the aircraft
-should be, then any error between the requested and actual rates will result in
-attitude error. So there is a feature called the integrator that continually
-sums the rate errors which effectively calculates the error in attitude.  The
-I gain is multiplied by the integrator and summed with the other outputs of the
-rate controller.  The integrator is limited by the :ref:`ATC_RAT_RLL_IMAX<ATC_RAT_RLL_IMAX__AC_AttitudeControl_Heli>` in roll and
-:ref:`ATC_RAT_PIT_IMAX<ATC_RAT_PIT_IMAX__AC_AttitudeControl_Heli>` in pitch.  When ground speed is less than 5 m/s, the
-integrator is leaked off (reduced at a specified rate) and another parameter, 
-:ref:`ATC_RAT_RLL_ILMI<ATC_RAT_RLL_ILMI>` and :ref:`ATC_RAT_PIT_ILMI <ATC_RAT_PIT_ILMI>`, only lets it leak off so much.  If the 
-ILMI, or integrator leak minimum, is zero then the integrator will not be 
-allowed to grow and the attitude will not be driven to exactly match the 
-software’s predicted attitude.  However, if this is non zero or large enough for
-attitude errors that may be encountered at low speeds and in a hover, then the 
-actual attitude will track the predicted attitude. The reason for the leak and 
-ILMI parameter is that a larger amount of integrator is needed for forward 
-flight. However, in a hover and in particular during air ground transition, 
-allowing large amounts of integrator can cause the aircraft to flip itself on
-its side.  So the integrator leak along with the leak minimum parameter keep 
-enough of the integrator to make it effective in keeping the attitudes matching
-but not so powerful to cause the aircraft to roll over.
+.. warning:: For making setting changes to traditional helicopters, users are reminded to use only the Full or Complete Parameter List, or the Heli SETUP page in your ground station software. **Do not use the Basic, Extended or Advanced Tuning pages that are designed for multi-rotor aircraft in Mission Planner.** These pages will make unwanted setting changes to traditional helicopters. And remember to write the changes to the flight controller after making them or they won't be saved!
+
+.. note:: If the reader is unfamiliar with PID control systems then reading :ref:`traditional-helicopter-control-system` is recommended.
 
 Initial Setup of Pitch and Roll Tuning Parameters
 =================================================
+
```

Comment:
In the Table below.  We should break out the ATC_RAT_XXX_FLTT and ATC_RAT_XXX_FLTD filter cut off frequencies as well.   Recommend that ATC_RAT_XXX_FLTD be set to zero and ATC_RAT_XXX_FLTT be set to 20 hz.  This is  defaulted to 10 hz currently but I think it is too low and adds to the time delay in the response.

Reply:
will do

Reply:
I will add the additional params and my current recommendations

Reply:
done